microsoft-edge/visual-studio-code/index.md

```diff
@@ -20,17 +20,11 @@ To acquire any of the extensions highlighted below, navigate to Extensions \(sel
 
 Search the Marketplace for the specific extension and choose **Install**.  
 
-:::image type="complex" source="./media/vscode-debugger-install.png" alt-text="Install the Debugger for Microsoft Edge Visual Studio Code extension" lightbox="./media/vscode-debugger-install.png":::
-   Install the **Debugger for Microsoft Edge** Visual Studio Code extension  
+:::image type="complex" source="./media/vscode-edge-tools-install.png" alt-text="Install the Microsoft Edge Tools for Visual Studio Code extension" lightbox="./media/vscode-edge-tools-install.png":::
+   Install the **Microsoft Edge Tools** Visual Studio Code extension  
```

Comment:
Change:
Hints appear as inline underlines and are summarized
to:
Hints appear as wavy underlines in the text editor, and are summarized